tests/chainer_tests/training_tests/triggers_tests/test_interval_trigger.py

```diff
@@ -76,5 +107,137 @@ def test_unaligned_epoch_interval_trigger(self):
         expected = [False, False, True, False, True, False, False]
         _test_trigger(self, updater, trigger, expected)
 
+    def test_unaligned_epoch_interval_stop_trigger(self):
+        updater = DummyUpdater(iters_per_epoch=2.5)
+        trigger = training.trigger.IntervalTrigger(1, 'epoch')
+        expected = [False, False, False, True]
+        _test_stop_trigger(self, updater, trigger, expected)
+
+
+class TestTinyEpochIntervalTrigger(unittest.TestCase):
+
+    def test_tiny_epoch_interval_trigger(self):
+        updater = DummyUpdater(iters_per_epoch=0.5)
+        trigger = training.trigger.IntervalTrigger(1, 'epoch')
+        expected = [True, True, True, True, True, True, True]
+        _test_trigger(self, updater, trigger, expected)
+
+    def test_tiny_epoch_interval_stop_trigger(self):
+        updater = DummyUpdater(iters_per_epoch=0.5)
+        trigger = training.trigger.IntervalTrigger(1, 'epoch')
+        expected = [False, True, True, True, True, True, True]
+        _test_stop_trigger(self, updater, trigger, expected)
+
+
+class TestResumedIterationIntervalTrigger(unittest.TestCase):
+
+    def test_resumed_iteration_interval_trigger(self):
+        updater = DummyUpdater(iters_per_epoch=5, initial_iteration=3)
+        trigger = training.trigger.IntervalTrigger(2, 'iteration')
+        expected = [True, False, True, False, True, False, True]
+        _test_trigger(self, updater, trigger, expected)
+
+    def test_resumed_iteration_interval_stop_trigger(self):
+        updater = DummyUpdater(iters_per_epoch=5, initial_iteration=3)
+        trigger = training.trigger.IntervalTrigger(2, 'iteration')
+        expected = [False, True]
+        _test_stop_trigger(self, updater, trigger, expected)
+
+
+class TestResumedEpochIntervalTrigger(unittest.TestCase):
+
+    def test_resumed_epoch_interval_trigger(self):
+        updater = DummyUpdater(iters_per_epoch=1, initial_iteration=4)
+        trigger = training.trigger.IntervalTrigger(3, 'epoch')
+        expected = [False, True, False, False, True, False]
+        _test_trigger(self, updater, trigger, expected)
```

Comment:
I see. I meant the test in which we do serialize the trigger to the temporary file, deserialize it and confirm it works correctly. Is it possible to write such a test?